tools/get-win-openssl.ps1

```diff
@@ -12,8 +12,8 @@ New-Item -ItemType Container $build_dir -ErrorAction Ignore
 $local_tgz = Join-Path $build_dir "openssl.tgz"
 
 # This is the path to the release static vs2019 x64 build of OpenSSL in bintray
-$conan_ssl_path = "_/openssl/1.1.1h/_/7098aea4e4f2247cc9b5dcaaa1ebddbe/package/a79a557254fabcb77849dd623fed97c9c5ab7651/141ef2c6711a254707ba1f7f4fd07ad4"
-$openssl_url = "https://dl.bintray.com/conan/conan-center/$conan_ssl_path/conan_package.tgz"
+$conan_ssl_path = "_/openssl/1.1.1h/_/986a09a7b68304213376088bd560b672/package/a79a557254fabcb77849dd623fed97c9c5ab7651/33ddaecbc11a8cef4eca0c21a9827f99"
```

Comment:
Was finally able to pin it down to a URL that appears to work correctly. Try this one:

```suggestion
$conan_ssl_path = "_/openssl/1.1.1h/_/8aee28bb7f844aa2e8e9f2b9e988027e/package/a79a557254fabcb77849dd623fed97c9c5ab7651/c200a0954463ed55a55d99809c0dba96"
```

Reply:
Still doesn't appear to have worked. https://vector-of-bool.visualstudio.com/dds/_build/results?buildId=1526&view=logs&jobId=dc6ba0e6-a07f-5909-ef10-dfbfdafc5a6e

Reply:
Feel free to make any commits you need to this PR to work out the issue; you don't need to wait for me to apply them if you don't want to.

Reply:
I'm losing my marbles on this one. I'll keep poking. Maybe I'll just vendor-in the OpenSSL tarball.

Reply:
I went ahead and added a commit that embeds the OpenSSL tgz. The Windows test still fails, but it seems to be an error with permissions to remove a file.